src/Minefield/Game/Board.cpp

```diff
@@ -21,63 +21,65 @@ TileType const* getTile(Coordinate const& tileCoordinate, TileMatrix const& matr
 
 namespace game
 {
-    using namespace utils::board;
+using namespace utils::board;
 
-    Board::Board(TileMatrix matrix, BoardDimensions boardDimensions) : mMatrix(matrix), mBoardDimensions(boardDimensions) {};
+Board::Board(TileMatrix matrix, BoardDimensions boardDimensions)
+: mMatrix(matrix)
+, mBoardDimensions(boardDimensions) {};
 
-    void Board::changeTileType(Coordinate const& tileCoordinate, TileType const& tileType)
+void Board::changeTileType(Coordinate const& tileCoordinate, TileType const& tileType)
+{
+    if (isValidCoordinate(tileCoordinate, mMatrix, mBoardDimensions))
     {
-        if (isValidCoordinate(tileCoordinate, mMatrix, mBoardDimensions))
-        {
-            mMatrix[tileCoordinate.x][tileCoordinate.y].tileType = tileType;
-        }
+        mMatrix[tileCoordinate.x][tileCoordinate.y].tileType = tileType;
     }
+}
 
-    void Board::print() const
+void Board::print() const
```

Comment:
#should be a free function, as you're not using any private field - yes, you're using `mMatrix` but you have a getter for it